client/ui/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The folder structure should be organized to clearly separate the client and server. Currently you have the backend and frontend under one directory structure "client", but client implies primarily frontend.

Maybe have something like:
```
<gQuant repo>
| - gquantui - rename current "client" to gquantui or something similar
|   | - client - rename current "ui" to just client or "gquantui-client"
|   |   | - .vscode - corresponds to current "<gQuant repo>/client/ui/.vscode"
|   |   \ - all the other files under "ui" currently
|   |
|   | - server - or similar name such as "gquantui-server"
|   |   | - .vcode - corresponds to current "<gQuant repo>/.vscode"
|   |   \ - server.py and other files for server

```